Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+        Returns:
+            Dict[str, Any]: Mapped dictionary.
+        """
+        return {'Simple String': 1, 'Regular Expression': 2}
+
+    @property
+    def request_type_api_to_user_mapper(self) -> Dict[Any, str]:
+        """Mapping the API output for request type to the user output
+        Returns:
+            Dict[str, Any]: Mapped dictionary.
+        """
+        return {1: 'Simple String', 2: 'Regular Expression'}
+
+    @property
+    def custom_whitelist_user_to_api_mapper(self) -> Dict[str, Any]:
+        """Mapping the user input for custom whitelist types to the API input
+        Returns:
+            Dict[str, Any]: Mapped dictionary.
+        """
+        return {'URL': 1, 'Parameter': 2, 'Cookie': 3}
+
+    @property
+    def custom_whitelist_api_to_user_mapper(self) -> Dict[Any, str]:
+        """Mapping the API output for custom whitelist types to the user output
+        Returns:
+            Dict[str, Any]: Mapped dictionary.
+        """
+        return {1: 'URL', 2: 'Parameter', 3: 'Cookie'}
+
+
+class ParserV2(Parser):
+
+    def create_output_headers(self, version: str, common_headers: List[str], v1_only_headers: List[str],
+                              v2_only_headers: List[str]) -> List[str]:
+        """Create headers for xsoar output.
+
+        Args:
+            version (str): Client version.
+            common_headers (List[str]): Common headers field for both versions.
+            v1_only_headers (List[str]): Headers for V1 only.
+            v2_only_headers (List[str]): Header for V2 only.
+
+        Returns:
+            List[str]: List of headers.
+        """
+        return common_headers + v2_only_headers
+
+    def parse_protected_hostname_group(self, protected_hostname_group: Dict[str, Any]) -> Dict[str, Any]:
+        """Parse for protected hostname group.
+
+        Args:
+            client (Client): Fortiweb VM client.
+            protected_hostname_group (Dict[str, Any]): A dictionary output from API.
+
+        Returns:
+            Dict[str, Any]: Parsed dictionary.
+        """
+        action_val = self.action_api_to_user_mapper[protected_hostname_group['default-action']]
+        group = {
+            'id': protected_hostname_group['name'],
+            'default_action': action_val,
+            'protected_hostname_count': protected_hostname_group['sz_host-list']
+        }
+        return group
+
+    def parse_protected_hostname_member(self, protected_hostname_member: Dict[str, Any]) -> Dict[str, Any]:
+        """Parse for protected hostname member.
+
+        Args:
+            client (Client): Fortiweb VM client.
+            protected_hostname_member (Dict[str, Any]): A dictionary output from API.
+
+        Returns:
+            Dict[str, Any]: Parsed disctionary.
+        """
+        group = {
+            'id': protected_hostname_member['id'],
+            'action': dict_safe_get(self.action_api_to_user_mapper, [protected_hostname_member['action']]),
```

Comment:
1. why use a safe get here? what's wrong with `self.action_api_to_user_mapper[protected_hostname_member['action']]`? Any chance an invalid value wil be provided?
2. If it fails, `None` is returned, does it make sense? 
3. don't use `dict_safe_get` with a list of size 1 in the 2nd argument, there's a built-in `.get` for that. please check the whole file.